src/test/java/org/kairosdb/core/properties/PropertiesValidatorTest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Dropbox
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kairosdb.core.properties;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.name.Names;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.kairosdb.core.exception.KairosDBException;
+import org.kairosdb.core.reporting.MetricReporterService;
+import org.kairosdb.datastore.cassandra.CassandraConfiguration;
+
+import java.util.Optional;
+
+/**
+ * Tests for {@link PropertiesValidator}.
+ *
+ * @author Max Wittek (https://github.com/wittekm)
+ */
+public final class PropertiesValidatorTest {
```

Comment:
nit. s/Ttl/TTL for testnames in this class

Reply:
Our convention if you use an acronym (which in general you should not) then it should be treated as a word due to its sufficiently common use to be used in code.